build/webpack.config.js

```diff
@@ -22,7 +22,8 @@ const webpackConfig = {
   resolve: {
     modules: [
       path.resolve(paths.client(), packageDir, 'src'),
-      'node_modules'
+      'node_modules',
```

Comment:
Is there a difference to:
path.resolve(paths.client(), packageDir, 'node_modules')